bentoml/migrations/versions/095fb029da39_init_tables.py

```diff
@@ -0,0 +1,33 @@
+"""init tables
+
+Revision ID: 095fb029da39
+Revises:
+Create Date: 2019-10-15 14:27:22.948583
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.engine.reflection import Inspector
+
+from bentoml.db import Base
+
+# revision identifiers, used by Alembic.
+
+revision = '095fb029da39'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
should be 2 revisions:

* 1: create a table matching  current schema created via "Base.metadata.create_all(bind=bind)" - we should check if table exists in this case due to we used to use just descriptive way for initializing a table
* 2: add update_at column as 2nd revision, from now on we use alembic to manage table schema